components/avatar/appearance.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Appearance
+page_title: Avatar Appearance
+description: Appearance settings of the Avatar for Blazor.
+slug: avatar-appearance
+tags: telerik,blazor,avatar,appearance
+published: True
+position: 10
+---
+
+# Appearance Settings
+
+You can control the appearance of the Avatar by setting the following attribute:
+
+* [Size](#size)
+* [Rounded](#rounded)
+* [FillMode](#fillmode)
+* [ThemeColor](#themecolor)
+
+You can use all of them together to achieve the desired appearance. This article will explain their effect one by one.
+
+## Size
+
+You can increase or decrease the size of the Avatar by setting the `Size` parameter to a member of the `Telerik.Blazor.ThemeConstants.Avatar.Size` class:
+
+| Class members | Manual declarations |
+|---------------|--------|
+| `Small`   |`sm`|
+| `Medium` (default value)  |`md`|
+| `Large`   |`lg`|
+
+> The default value of the `Size` will take precedence over the values of the `Width` and `Height` parameters. Set the `Size` to an empty `string` to apply your custom `Width` and `Height`.
```

Comment:
I would challenge the developer for this decision. Why it's necessary?

Reply:
To be discussed with the development team.